### PR TITLE
tabs show onPress item

### PR DESCRIPTION
**Pull request by phproot**
tabs show onPress item

Examples:

```
<Scene key="Modal" component={Modal}>
        <Scene tabs={true}  …  >
            <Scene key="Launch"  onPress={()=>Actions._Launch()} …/>
            <Scene key="Contacts" … />
            <Scene key="Me"  … />
        </Scene>
    <Scene key="_Launch" component={Launch}/>
</Scene>

```
look like mac os Launchpad

### CODE REVIEW

**Comment by joenoon** [comment]
Comment from other PR: https://github.com/aksonov/react-native-router-flux/pull/827#issuecomment-226388281

File: src/DefaultRenderer.js [hunk 1]
```diff
@@ -211,6 +211,11 @@ export default class DefaultRenderer extends Component {
     }
 
     if (SceneComponent) {
+      if (navigationState.tabs || !navigationState.parent) {
```
Comment (joenoon) [inline src/DefaultRenderer.js]: What is the need for having a different code path here?  Is there something in the Example app I can check?  Does this change break any of the Example app pages?

Comment (phproot) [inline src/DefaultRenderer.js]: sorry,my bad.

Comment (phproot) [inline src/DefaultRenderer.js]: if sceneStyle set paddingTop、alignItems... 
modal、tabs、NavigationRootView will inherit these attributes。

Comment (joenoon) [inline src/DefaultRenderer.js]: @phproot no problem! you can probably use getSceneStyle to account for the hideNavBar/hideTabBar props like in the Example app, and not hard-code a marginTop in sceneStyle.  Check the Example app that does exactly this.

That 3D screenshot is great.  Is that Reveal.app or something else?

**Comment by joenoon** [comment]
@aksonov this seems ok to me, except I'm not sure if there could be unexpected problems with the `condition`.  That whole part of Actions is a little unclear to me.

File: src/DefaultRenderer.js (hunk 1 above)
Comment (phproot) [inline src/DefaultRenderer.js]: @joenoon is xcode 
Debug -> View Debugging -> Capture View Hierarchy

**Comment by aksonov** [comment]
@alextkd Is it OK to merge?

**Comment by phproot** [comment]
What are Can we merge?

**Comment by alextkd** [comment]
Looks good to me. I don't see any problems it can cause.

**Pull request merged**